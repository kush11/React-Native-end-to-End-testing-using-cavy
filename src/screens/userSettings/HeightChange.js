import React, { Component } from 'react';
import {
  Platform,
  StyleSheet,
  Text,
  View,
  TouchableOpacity, PickerIOS
} from 'react-native';
import { connect } from 'react-redux';
import Picker from 'react-native-wheel-picker';
import { updateHeight, updateBmi } from '../../store/actions/index';
import { BASE_URL, headers } from '../../api/config/Config';
import CalculateBmi from '../../components/utility/bmi/Bmi';
import { regularButtonFont, fontMaker } from '../../components/utility/fonts/FontMaker';


const PickerItemIOS = PickerIOS.Item;

const mapStateToProps = state => ({
  name1: state.User.name,
  uHeight: state.User.height,
  uWeight: state.User.weight,
});
const mapDispatchToProps = dispatch => ({
  updatedHeight: height => dispatch(updateHeight(height)),
  updatedBmi: bmi => dispatch(updateBmi(bmi))
});
const PickerItem = Picker.Item;

export class HeightChange extends Component {
  constructor(props) {
    super(props);
    const { uHeight } = this.props;
    this.state = {
      selectedItem: uHeight === '' ? 0 : uHeight,
      itemList: []

    };
    const { itemList } = this.state;
    for (let i = 0; i <= 300; i += 1) {
      itemList[i] = i;
    }
  }

  onPickerSelect(index) {
    this.setState({ selectedItem: index });
  }

  onAddItem = () => {
    const {
      name1, uWeight, navigation, updatedHeight, updatedBmi
    } = this.props;
    const { itemList, selectedItem } = this.state;
    fetch(`${BASE_URL}/api/userData`, {
      method: 'POST',
      headers,
      body: JSON.stringify({
        name: name1,
        height: itemList[selectedItem]
      })
    }).then(response => response.json(),
      updatedHeight(itemList[selectedItem]))
      .then(updatedBmi(CalculateBmi(itemList[selectedItem], uWeight)))
      .then(navigation.navigate('PersonalSetting'))
      .catch(() => { });
  }

  render() {
    const { selectedItem, itemList } = this.state;
    return (
      <View style={styles.container}>
        <Text style={styles.welcome}>
          Please select
        </Text>

        {Platform.OS === 'ios'
          ? (
            <PickerIOS
              style={{
                marginTop: 20, alignSelf: 'center', width: '100%', height: 180, backgroundColor: '#dcdcdc',
              }}
              selectedValue={selectedItem}
              itemStyle={{ color: 'black', fontSize: 26 }}
              onValueChange={index => this.onPickerSelect(index)}
            >
              {itemList.map((value, i) => (
                <PickerItemIOS
                  key={`number${value.toString()}`}
                  value={i}
                  label={value.toString()}
                />
              ))}
            </PickerIOS>
          )
          : (
            <Picker
              style={{
                width: '100%',
                height: 180,
                justifyContent: 'center',
                alignItems: 'center',
                backgroundColor: '#eee',
                marginTop: 20
              }}
              selectedValue={selectedItem}
              itemStyle={{ color: 'black', fontSize: 26 }}
              onValueChange={index => this.onPickerSelect(index)}
            >
              {itemList.map((value, i) => (
                <PickerItem label={value.toString()} value={i} key={`number${value.toString()}`} />
              ))}
            </Picker>
          )}
        <Text style={{
          margin: 20,
          color: '#000000',
          fontSize: 20,
          justifyContent: 'center',
          marginTop: 30,
          alignItems: 'center',
          alignSelf: 'center',
          ...titleFontStyle
        }}
        >
          Selected :
          {' '}
          {itemList[selectedItem]}
          {' '}
          cm
        </Text>
        <TouchableOpacity style={styles.setBmiBtn} onPress={this.onAddItem}>
          <Text style={styles.textWhite}>{'Set Height'.toUpperCase()}</Text>
        </TouchableOpacity>
      </View>
    );
  }
}

const titleFontStyle = fontMaker({ family: 'Montserrat', weight: 'Regular' });
const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'white'
  },
  setBmiBtn: {
    backgroundColor: '#41ab3e',
    alignItems: 'center',
    marginVertical: 15,
    paddingVertical: 12,
    borderRadius: 8,
    marginRight: 60,
    marginLeft: 60

  },
  textWhite: {
    color: '#ffffff',
    fontSize: 15,
    ...regularButtonFont
  },
  welcome: {
    fontSize: 16,
    textAlign: 'center',
    marginTop: 15,
    color: 'black',
    ...titleFontStyle
  },
});

export default connect(mapStateToProps, mapDispatchToProps)(HeightChange);
