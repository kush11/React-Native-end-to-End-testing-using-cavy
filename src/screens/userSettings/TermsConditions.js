import React from 'react';
import { View, StyleSheet, Text } from 'react-native';
import { fontMaker } from '../../components/utility/fonts/FontMaker';

const termsConditions = (props) => {
  const { children } = props;
  return (
    <View>
      <Text style={styles.listText}>
        {children}
      </Text>
    </View>
  );
};

const termsFontStyle = fontMaker({ family: 'OpenSans', weight: 'Regular' });
const styles = StyleSheet.create({
  container: { flex: 1 },
  list: { flexDirection: 'row' },
  listText: {
    fontSize: 18,
    textAlign: 'center',
    color: 'black',
    ...termsFontStyle
  },
});
export default termsConditions;
