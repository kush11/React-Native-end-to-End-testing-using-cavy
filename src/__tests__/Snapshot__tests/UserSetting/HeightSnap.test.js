import 'react-native';
import React from 'react';
import Renderer from 'react-test-renderer';
import { HeightChange } from '../../../screens/userSettings/HeightChange';

test('BiologicalAgeScore snapshot', () => {
  const snap = Renderer.create(
    <HeightChange />
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
