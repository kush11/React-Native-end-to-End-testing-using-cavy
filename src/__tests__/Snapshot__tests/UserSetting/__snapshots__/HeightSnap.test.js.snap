// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BiologicalAgeScore snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontFamily": "Montserrat",
        "fontSize": 16,
        "fontStyle": "normal",
        "fontWeight": "400",
        "marginTop": 15,
        "textAlign": "center",
      }
    }
  >
    Please select
  </Text>
  <View
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "#dcdcdc",
        "height": 180,
        "marginTop": 20,
        "width": "100%",
      }
    }
  >
    <RCTPicker
      items={
        Array [
          Object {
            "label": "0",
            "textColor": undefined,
            "value": 0,
          },
          Object {
            "label": "1",
            "textColor": undefined,
            "value": 1,
          },
          Object {
            "label": "2",
            "textColor": undefined,
            "value": 2,
          },
          Object {
            "label": "3",
            "textColor": undefined,
            "value": 3,
          },
          Object {
            "label": "4",
            "textColor": undefined,
            "value": 4,
          },
          Object {
            "label": "5",
            "textColor": undefined,
            "value": 5,
          },
          Object {
            "label": "6",
            "textColor": undefined,
            "value": 6,
          },
          Object {
            "label": "7",
            "textColor": undefined,
            "value": 7,
          },
          Object {
            "label": "8",
            "textColor": undefined,
            "value": 8,
          },
          Object {
            "label": "9",
            "textColor": undefined,
            "value": 9,
          },
          Object {
            "label": "10",
            "textColor": undefined,
            "value": 10,
          },
          Object {
            "label": "11",
            "textColor": undefined,
            "value": 11,
          },
          Object {
            "label": "12",
            "textColor": undefined,
            "value": 12,
          },
          Object {
            "label": "13",
            "textColor": undefined,
            "value": 13,
          },
          Object {
            "label": "14",
            "textColor": undefined,
            "value": 14,
          },
          Object {
            "label": "15",
            "textColor": undefined,
            "value": 15,
          },
          Object {
            "label": "16",
            "textColor": undefined,
            "value": 16,
          },
          Object {
            "label": "17",
            "textColor": undefined,
            "value": 17,
          },
          Object {
            "label": "18",
            "textColor": undefined,
            "value": 18,
          },
          Object {
            "label": "19",
            "textColor": undefined,
            "value": 19,
          },
          Object {
            "label": "20",
            "textColor": undefined,
            "value": 20,
          },
          Object {
            "label": "21",
            "textColor": undefined,
            "value": 21,
          },
          Object {
            "label": "22",
            "textColor": undefined,
            "value": 22,
          },
          Object {
            "label": "23",
            "textColor": undefined,
            "value": 23,
          },
          Object {
            "label": "24",
            "textColor": undefined,
            "value": 24,
          },
          Object {
            "label": "25",
            "textColor": undefined,
            "value": 25,
          },
          Object {
            "label": "26",
            "textColor": undefined,
            "value": 26,
          },
          Object {
            "label": "27",
            "textColor": undefined,
            "value": 27,
          },
          Object {
            "label": "28",
            "textColor": undefined,
            "value": 28,
          },
          Object {
            "label": "29",
            "textColor": undefined,
            "value": 29,
          },
          Object {
            "label": "30",
            "textColor": undefined,
            "value": 30,
          },
          Object {
            "label": "31",
            "textColor": undefined,
            "value": 31,
          },
          Object {
            "label": "32",
            "textColor": undefined,
            "value": 32,
          },
          Object {
            "label": "33",
            "textColor": undefined,
            "value": 33,
          },
          Object {
            "label": "34",
            "textColor": undefined,
            "value": 34,
          },
          Object {
            "label": "35",
            "textColor": undefined,
            "value": 35,
          },
          Object {
            "label": "36",
            "textColor": undefined,
            "value": 36,
          },
          Object {
            "label": "37",
            "textColor": undefined,
            "value": 37,
          },
          Object {
            "label": "38",
            "textColor": undefined,
            "value": 38,
          },
          Object {
            "label": "39",
            "textColor": undefined,
            "value": 39,
          },
          Object {
            "label": "40",
            "textColor": undefined,
            "value": 40,
          },
          Object {
            "label": "41",
            "textColor": undefined,
            "value": 41,
          },
          Object {
            "label": "42",
            "textColor": undefined,
            "value": 42,
          },
          Object {
            "label": "43",
            "textColor": undefined,
            "value": 43,
          },
          Object {
            "label": "44",
            "textColor": undefined,
            "value": 44,
          },
          Object {
            "label": "45",
            "textColor": undefined,
            "value": 45,
          },
          Object {
            "label": "46",
            "textColor": undefined,
            "value": 46,
          },
          Object {
            "label": "47",
            "textColor": undefined,
            "value": 47,
          },
          Object {
            "label": "48",
            "textColor": undefined,
            "value": 48,
          },
          Object {
            "label": "49",
            "textColor": undefined,
            "value": 49,
          },
          Object {
            "label": "50",
            "textColor": undefined,
            "value": 50,
          },
          Object {
            "label": "51",
            "textColor": undefined,
            "value": 51,
          },
          Object {
            "label": "52",
            "textColor": undefined,
            "value": 52,
          },
          Object {
            "label": "53",
            "textColor": undefined,
            "value": 53,
          },
          Object {
            "label": "54",
            "textColor": undefined,
            "value": 54,
          },
          Object {
            "label": "55",
            "textColor": undefined,
            "value": 55,
          },
          Object {
            "label": "56",
            "textColor": undefined,
            "value": 56,
          },
          Object {
            "label": "57",
            "textColor": undefined,
            "value": 57,
          },
          Object {
            "label": "58",
            "textColor": undefined,
            "value": 58,
          },
          Object {
            "label": "59",
            "textColor": undefined,
            "value": 59,
          },
          Object {
            "label": "60",
            "textColor": undefined,
            "value": 60,
          },
          Object {
            "label": "61",
            "textColor": undefined,
            "value": 61,
          },
          Object {
            "label": "62",
            "textColor": undefined,
            "value": 62,
          },
          Object {
            "label": "63",
            "textColor": undefined,
            "value": 63,
          },
          Object {
            "label": "64",
            "textColor": undefined,
            "value": 64,
          },
          Object {
            "label": "65",
            "textColor": undefined,
            "value": 65,
          },
          Object {
            "label": "66",
            "textColor": undefined,
            "value": 66,
          },
          Object {
            "label": "67",
            "textColor": undefined,
            "value": 67,
          },
          Object {
            "label": "68",
            "textColor": undefined,
            "value": 68,
          },
          Object {
            "label": "69",
            "textColor": undefined,
            "value": 69,
          },
          Object {
            "label": "70",
            "textColor": undefined,
            "value": 70,
          },
          Object {
            "label": "71",
            "textColor": undefined,
            "value": 71,
          },
          Object {
            "label": "72",
            "textColor": undefined,
            "value": 72,
          },
          Object {
            "label": "73",
            "textColor": undefined,
            "value": 73,
          },
          Object {
            "label": "74",
            "textColor": undefined,
            "value": 74,
          },
          Object {
            "label": "75",
            "textColor": undefined,
            "value": 75,
          },
          Object {
            "label": "76",
            "textColor": undefined,
            "value": 76,
          },
          Object {
            "label": "77",
            "textColor": undefined,
            "value": 77,
          },
          Object {
            "label": "78",
            "textColor": undefined,
            "value": 78,
          },
          Object {
            "label": "79",
            "textColor": undefined,
            "value": 79,
          },
          Object {
            "label": "80",
            "textColor": undefined,
            "value": 80,
          },
          Object {
            "label": "81",
            "textColor": undefined,
            "value": 81,
          },
          Object {
            "label": "82",
            "textColor": undefined,
            "value": 82,
          },
          Object {
            "label": "83",
            "textColor": undefined,
            "value": 83,
          },
          Object {
            "label": "84",
            "textColor": undefined,
            "value": 84,
          },
          Object {
            "label": "85",
            "textColor": undefined,
            "value": 85,
          },
          Object {
            "label": "86",
            "textColor": undefined,
            "value": 86,
          },
          Object {
            "label": "87",
            "textColor": undefined,
            "value": 87,
          },
          Object {
            "label": "88",
            "textColor": undefined,
            "value": 88,
          },
          Object {
            "label": "89",
            "textColor": undefined,
            "value": 89,
          },
          Object {
            "label": "90",
            "textColor": undefined,
            "value": 90,
          },
          Object {
            "label": "91",
            "textColor": undefined,
            "value": 91,
          },
          Object {
            "label": "92",
            "textColor": undefined,
            "value": 92,
          },
          Object {
            "label": "93",
            "textColor": undefined,
            "value": 93,
          },
          Object {
            "label": "94",
            "textColor": undefined,
            "value": 94,
          },
          Object {
            "label": "95",
            "textColor": undefined,
            "value": 95,
          },
          Object {
            "label": "96",
            "textColor": undefined,
            "value": 96,
          },
          Object {
            "label": "97",
            "textColor": undefined,
            "value": 97,
          },
          Object {
            "label": "98",
            "textColor": undefined,
            "value": 98,
          },
          Object {
            "label": "99",
            "textColor": undefined,
            "value": 99,
          },
          Object {
            "label": "100",
            "textColor": undefined,
            "value": 100,
          },
          Object {
            "label": "101",
            "textColor": undefined,
            "value": 101,
          },
          Object {
            "label": "102",
            "textColor": undefined,
            "value": 102,
          },
          Object {
            "label": "103",
            "textColor": undefined,
            "value": 103,
          },
          Object {
            "label": "104",
            "textColor": undefined,
            "value": 104,
          },
          Object {
            "label": "105",
            "textColor": undefined,
            "value": 105,
          },
          Object {
            "label": "106",
            "textColor": undefined,
            "value": 106,
          },
          Object {
            "label": "107",
            "textColor": undefined,
            "value": 107,
          },
          Object {
            "label": "108",
            "textColor": undefined,
            "value": 108,
          },
          Object {
            "label": "109",
            "textColor": undefined,
            "value": 109,
          },
          Object {
            "label": "110",
            "textColor": undefined,
            "value": 110,
          },
          Object {
            "label": "111",
            "textColor": undefined,
            "value": 111,
          },
          Object {
            "label": "112",
            "textColor": undefined,
            "value": 112,
          },
          Object {
            "label": "113",
            "textColor": undefined,
            "value": 113,
          },
          Object {
            "label": "114",
            "textColor": undefined,
            "value": 114,
          },
          Object {
            "label": "115",
            "textColor": undefined,
            "value": 115,
          },
          Object {
            "label": "116",
            "textColor": undefined,
            "value": 116,
          },
          Object {
            "label": "117",
            "textColor": undefined,
            "value": 117,
          },
          Object {
            "label": "118",
            "textColor": undefined,
            "value": 118,
          },
          Object {
            "label": "119",
            "textColor": undefined,
            "value": 119,
          },
          Object {
            "label": "120",
            "textColor": undefined,
            "value": 120,
          },
          Object {
            "label": "121",
            "textColor": undefined,
            "value": 121,
          },
          Object {
            "label": "122",
            "textColor": undefined,
            "value": 122,
          },
          Object {
            "label": "123",
            "textColor": undefined,
            "value": 123,
          },
          Object {
            "label": "124",
            "textColor": undefined,
            "value": 124,
          },
          Object {
            "label": "125",
            "textColor": undefined,
            "value": 125,
          },
          Object {
            "label": "126",
            "textColor": undefined,
            "value": 126,
          },
          Object {
            "label": "127",
            "textColor": undefined,
            "value": 127,
          },
          Object {
            "label": "128",
            "textColor": undefined,
            "value": 128,
          },
          Object {
            "label": "129",
            "textColor": undefined,
            "value": 129,
          },
          Object {
            "label": "130",
            "textColor": undefined,
            "value": 130,
          },
          Object {
            "label": "131",
            "textColor": undefined,
            "value": 131,
          },
          Object {
            "label": "132",
            "textColor": undefined,
            "value": 132,
          },
          Object {
            "label": "133",
            "textColor": undefined,
            "value": 133,
          },
          Object {
            "label": "134",
            "textColor": undefined,
            "value": 134,
          },
          Object {
            "label": "135",
            "textColor": undefined,
            "value": 135,
          },
          Object {
            "label": "136",
            "textColor": undefined,
            "value": 136,
          },
          Object {
            "label": "137",
            "textColor": undefined,
            "value": 137,
          },
          Object {
            "label": "138",
            "textColor": undefined,
            "value": 138,
          },
          Object {
            "label": "139",
            "textColor": undefined,
            "value": 139,
          },
          Object {
            "label": "140",
            "textColor": undefined,
            "value": 140,
          },
          Object {
            "label": "141",
            "textColor": undefined,
            "value": 141,
          },
          Object {
            "label": "142",
            "textColor": undefined,
            "value": 142,
          },
          Object {
            "label": "143",
            "textColor": undefined,
            "value": 143,
          },
          Object {
            "label": "144",
            "textColor": undefined,
            "value": 144,
          },
          Object {
            "label": "145",
            "textColor": undefined,
            "value": 145,
          },
          Object {
            "label": "146",
            "textColor": undefined,
            "value": 146,
          },
          Object {
            "label": "147",
            "textColor": undefined,
            "value": 147,
          },
          Object {
            "label": "148",
            "textColor": undefined,
            "value": 148,
          },
          Object {
            "label": "149",
            "textColor": undefined,
            "value": 149,
          },
          Object {
            "label": "150",
            "textColor": undefined,
            "value": 150,
          },
          Object {
            "label": "151",
            "textColor": undefined,
            "value": 151,
          },
          Object {
            "label": "152",
            "textColor": undefined,
            "value": 152,
          },
          Object {
            "label": "153",
            "textColor": undefined,
            "value": 153,
          },
          Object {
            "label": "154",
            "textColor": undefined,
            "value": 154,
          },
          Object {
            "label": "155",
            "textColor": undefined,
            "value": 155,
          },
          Object {
            "label": "156",
            "textColor": undefined,
            "value": 156,
          },
          Object {
            "label": "157",
            "textColor": undefined,
            "value": 157,
          },
          Object {
            "label": "158",
            "textColor": undefined,
            "value": 158,
          },
          Object {
            "label": "159",
            "textColor": undefined,
            "value": 159,
          },
          Object {
            "label": "160",
            "textColor": undefined,
            "value": 160,
          },
          Object {
            "label": "161",
            "textColor": undefined,
            "value": 161,
          },
          Object {
            "label": "162",
            "textColor": undefined,
            "value": 162,
          },
          Object {
            "label": "163",
            "textColor": undefined,
            "value": 163,
          },
          Object {
            "label": "164",
            "textColor": undefined,
            "value": 164,
          },
          Object {
            "label": "165",
            "textColor": undefined,
            "value": 165,
          },
          Object {
            "label": "166",
            "textColor": undefined,
            "value": 166,
          },
          Object {
            "label": "167",
            "textColor": undefined,
            "value": 167,
          },
          Object {
            "label": "168",
            "textColor": undefined,
            "value": 168,
          },
          Object {
            "label": "169",
            "textColor": undefined,
            "value": 169,
          },
          Object {
            "label": "170",
            "textColor": undefined,
            "value": 170,
          },
          Object {
            "label": "171",
            "textColor": undefined,
            "value": 171,
          },
          Object {
            "label": "172",
            "textColor": undefined,
            "value": 172,
          },
          Object {
            "label": "173",
            "textColor": undefined,
            "value": 173,
          },
          Object {
            "label": "174",
            "textColor": undefined,
            "value": 174,
          },
          Object {
            "label": "175",
            "textColor": undefined,
            "value": 175,
          },
          Object {
            "label": "176",
            "textColor": undefined,
            "value": 176,
          },
          Object {
            "label": "177",
            "textColor": undefined,
            "value": 177,
          },
          Object {
            "label": "178",
            "textColor": undefined,
            "value": 178,
          },
          Object {
            "label": "179",
            "textColor": undefined,
            "value": 179,
          },
          Object {
            "label": "180",
            "textColor": undefined,
            "value": 180,
          },
          Object {
            "label": "181",
            "textColor": undefined,
            "value": 181,
          },
          Object {
            "label": "182",
            "textColor": undefined,
            "value": 182,
          },
          Object {
            "label": "183",
            "textColor": undefined,
            "value": 183,
          },
          Object {
            "label": "184",
            "textColor": undefined,
            "value": 184,
          },
          Object {
            "label": "185",
            "textColor": undefined,
            "value": 185,
          },
          Object {
            "label": "186",
            "textColor": undefined,
            "value": 186,
          },
          Object {
            "label": "187",
            "textColor": undefined,
            "value": 187,
          },
          Object {
            "label": "188",
            "textColor": undefined,
            "value": 188,
          },
          Object {
            "label": "189",
            "textColor": undefined,
            "value": 189,
          },
          Object {
            "label": "190",
            "textColor": undefined,
            "value": 190,
          },
          Object {
            "label": "191",
            "textColor": undefined,
            "value": 191,
          },
          Object {
            "label": "192",
            "textColor": undefined,
            "value": 192,
          },
          Object {
            "label": "193",
            "textColor": undefined,
            "value": 193,
          },
          Object {
            "label": "194",
            "textColor": undefined,
            "value": 194,
          },
          Object {
            "label": "195",
            "textColor": undefined,
            "value": 195,
          },
          Object {
            "label": "196",
            "textColor": undefined,
            "value": 196,
          },
          Object {
            "label": "197",
            "textColor": undefined,
            "value": 197,
          },
          Object {
            "label": "198",
            "textColor": undefined,
            "value": 198,
          },
          Object {
            "label": "199",
            "textColor": undefined,
            "value": 199,
          },
          Object {
            "label": "200",
            "textColor": undefined,
            "value": 200,
          },
          Object {
            "label": "201",
            "textColor": undefined,
            "value": 201,
          },
          Object {
            "label": "202",
            "textColor": undefined,
            "value": 202,
          },
          Object {
            "label": "203",
            "textColor": undefined,
            "value": 203,
          },
          Object {
            "label": "204",
            "textColor": undefined,
            "value": 204,
          },
          Object {
            "label": "205",
            "textColor": undefined,
            "value": 205,
          },
          Object {
            "label": "206",
            "textColor": undefined,
            "value": 206,
          },
          Object {
            "label": "207",
            "textColor": undefined,
            "value": 207,
          },
          Object {
            "label": "208",
            "textColor": undefined,
            "value": 208,
          },
          Object {
            "label": "209",
            "textColor": undefined,
            "value": 209,
          },
          Object {
            "label": "210",
            "textColor": undefined,
            "value": 210,
          },
          Object {
            "label": "211",
            "textColor": undefined,
            "value": 211,
          },
          Object {
            "label": "212",
            "textColor": undefined,
            "value": 212,
          },
          Object {
            "label": "213",
            "textColor": undefined,
            "value": 213,
          },
          Object {
            "label": "214",
            "textColor": undefined,
            "value": 214,
          },
          Object {
            "label": "215",
            "textColor": undefined,
            "value": 215,
          },
          Object {
            "label": "216",
            "textColor": undefined,
            "value": 216,
          },
          Object {
            "label": "217",
            "textColor": undefined,
            "value": 217,
          },
          Object {
            "label": "218",
            "textColor": undefined,
            "value": 218,
          },
          Object {
            "label": "219",
            "textColor": undefined,
            "value": 219,
          },
          Object {
            "label": "220",
            "textColor": undefined,
            "value": 220,
          },
          Object {
            "label": "221",
            "textColor": undefined,
            "value": 221,
          },
          Object {
            "label": "222",
            "textColor": undefined,
            "value": 222,
          },
          Object {
            "label": "223",
            "textColor": undefined,
            "value": 223,
          },
          Object {
            "label": "224",
            "textColor": undefined,
            "value": 224,
          },
          Object {
            "label": "225",
            "textColor": undefined,
            "value": 225,
          },
          Object {
            "label": "226",
            "textColor": undefined,
            "value": 226,
          },
          Object {
            "label": "227",
            "textColor": undefined,
            "value": 227,
          },
          Object {
            "label": "228",
            "textColor": undefined,
            "value": 228,
          },
          Object {
            "label": "229",
            "textColor": undefined,
            "value": 229,
          },
          Object {
            "label": "230",
            "textColor": undefined,
            "value": 230,
          },
          Object {
            "label": "231",
            "textColor": undefined,
            "value": 231,
          },
          Object {
            "label": "232",
            "textColor": undefined,
            "value": 232,
          },
          Object {
            "label": "233",
            "textColor": undefined,
            "value": 233,
          },
          Object {
            "label": "234",
            "textColor": undefined,
            "value": 234,
          },
          Object {
            "label": "235",
            "textColor": undefined,
            "value": 235,
          },
          Object {
            "label": "236",
            "textColor": undefined,
            "value": 236,
          },
          Object {
            "label": "237",
            "textColor": undefined,
            "value": 237,
          },
          Object {
            "label": "238",
            "textColor": undefined,
            "value": 238,
          },
          Object {
            "label": "239",
            "textColor": undefined,
            "value": 239,
          },
          Object {
            "label": "240",
            "textColor": undefined,
            "value": 240,
          },
          Object {
            "label": "241",
            "textColor": undefined,
            "value": 241,
          },
          Object {
            "label": "242",
            "textColor": undefined,
            "value": 242,
          },
          Object {
            "label": "243",
            "textColor": undefined,
            "value": 243,
          },
          Object {
            "label": "244",
            "textColor": undefined,
            "value": 244,
          },
          Object {
            "label": "245",
            "textColor": undefined,
            "value": 245,
          },
          Object {
            "label": "246",
            "textColor": undefined,
            "value": 246,
          },
          Object {
            "label": "247",
            "textColor": undefined,
            "value": 247,
          },
          Object {
            "label": "248",
            "textColor": undefined,
            "value": 248,
          },
          Object {
            "label": "249",
            "textColor": undefined,
            "value": 249,
          },
          Object {
            "label": "250",
            "textColor": undefined,
            "value": 250,
          },
          Object {
            "label": "251",
            "textColor": undefined,
            "value": 251,
          },
          Object {
            "label": "252",
            "textColor": undefined,
            "value": 252,
          },
          Object {
            "label": "253",
            "textColor": undefined,
            "value": 253,
          },
          Object {
            "label": "254",
            "textColor": undefined,
            "value": 254,
          },
          Object {
            "label": "255",
            "textColor": undefined,
            "value": 255,
          },
          Object {
            "label": "256",
            "textColor": undefined,
            "value": 256,
          },
          Object {
            "label": "257",
            "textColor": undefined,
            "value": 257,
          },
          Object {
            "label": "258",
            "textColor": undefined,
            "value": 258,
          },
          Object {
            "label": "259",
            "textColor": undefined,
            "value": 259,
          },
          Object {
            "label": "260",
            "textColor": undefined,
            "value": 260,
          },
          Object {
            "label": "261",
            "textColor": undefined,
            "value": 261,
          },
          Object {
            "label": "262",
            "textColor": undefined,
            "value": 262,
          },
          Object {
            "label": "263",
            "textColor": undefined,
            "value": 263,
          },
          Object {
            "label": "264",
            "textColor": undefined,
            "value": 264,
          },
          Object {
            "label": "265",
            "textColor": undefined,
            "value": 265,
          },
          Object {
            "label": "266",
            "textColor": undefined,
            "value": 266,
          },
          Object {
            "label": "267",
            "textColor": undefined,
            "value": 267,
          },
          Object {
            "label": "268",
            "textColor": undefined,
            "value": 268,
          },
          Object {
            "label": "269",
            "textColor": undefined,
            "value": 269,
          },
          Object {
            "label": "270",
            "textColor": undefined,
            "value": 270,
          },
          Object {
            "label": "271",
            "textColor": undefined,
            "value": 271,
          },
          Object {
            "label": "272",
            "textColor": undefined,
            "value": 272,
          },
          Object {
            "label": "273",
            "textColor": undefined,
            "value": 273,
          },
          Object {
            "label": "274",
            "textColor": undefined,
            "value": 274,
          },
          Object {
            "label": "275",
            "textColor": undefined,
            "value": 275,
          },
          Object {
            "label": "276",
            "textColor": undefined,
            "value": 276,
          },
          Object {
            "label": "277",
            "textColor": undefined,
            "value": 277,
          },
          Object {
            "label": "278",
            "textColor": undefined,
            "value": 278,
          },
          Object {
            "label": "279",
            "textColor": undefined,
            "value": 279,
          },
          Object {
            "label": "280",
            "textColor": undefined,
            "value": 280,
          },
          Object {
            "label": "281",
            "textColor": undefined,
            "value": 281,
          },
          Object {
            "label": "282",
            "textColor": undefined,
            "value": 282,
          },
          Object {
            "label": "283",
            "textColor": undefined,
            "value": 283,
          },
          Object {
            "label": "284",
            "textColor": undefined,
            "value": 284,
          },
          Object {
            "label": "285",
            "textColor": undefined,
            "value": 285,
          },
          Object {
            "label": "286",
            "textColor": undefined,
            "value": 286,
          },
          Object {
            "label": "287",
            "textColor": undefined,
            "value": 287,
          },
          Object {
            "label": "288",
            "textColor": undefined,
            "value": 288,
          },
          Object {
            "label": "289",
            "textColor": undefined,
            "value": 289,
          },
          Object {
            "label": "290",
            "textColor": undefined,
            "value": 290,
          },
          Object {
            "label": "291",
            "textColor": undefined,
            "value": 291,
          },
          Object {
            "label": "292",
            "textColor": undefined,
            "value": 292,
          },
          Object {
            "label": "293",
            "textColor": undefined,
            "value": 293,
          },
          Object {
            "label": "294",
            "textColor": undefined,
            "value": 294,
          },
          Object {
            "label": "295",
            "textColor": undefined,
            "value": 295,
          },
          Object {
            "label": "296",
            "textColor": undefined,
            "value": 296,
          },
          Object {
            "label": "297",
            "textColor": undefined,
            "value": 297,
          },
          Object {
            "label": "298",
            "textColor": undefined,
            "value": 298,
          },
          Object {
            "label": "299",
            "textColor": undefined,
            "value": 299,
          },
          Object {
            "label": "300",
            "textColor": undefined,
            "value": 300,
          },
        ]
      }
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      selectedIndex={0}
      style={
        Array [
          Object {
            "height": 216,
          },
          Object {
            "color": "black",
            "fontSize": 26,
          },
        ]
      }
    />
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "center",
        "color": "#000000",
        "fontFamily": "Montserrat",
        "fontSize": 20,
        "fontStyle": "normal",
        "fontWeight": "400",
        "justifyContent": "center",
        "margin": 20,
        "marginTop": 30,
      }
    }
  >
    Selected :
     
     
    cm
  </Text>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#41ab3e",
        "borderRadius": 8,
        "marginLeft": 60,
        "marginRight": 60,
        "marginVertical": 15,
        "opacity": 1,
        "paddingVertical": 12,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "fontFamily": "OpenSans-Regular",
          "fontSize": 15,
        }
      }
    >
      SET HEIGHT
    </Text>
  </View>
</View>
`;
